packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx

```diff
@@ -261,14 +266,19 @@ UserSettingsMenuBody.displayName = 'UserSettingsMenuBody';
 
 const UserSettingsMenu = (props: Props) => {
   const intl = useIntl();
+  const menuLabelId = useFieldId(undefined, menuLabelSequentialId);
 
   return (
     <div data-test="user-settings-menu">
-      <Downshift stateReducer={stateReducer}>
+      <AccessibleHidden>
+        <span id={menuLabelId}>
```

Comment:
There is only one user menu in the page, so we can provide a fixed id here, no need for generating a random one.

Reply:
Updated in this commit: eb558206564c5ed2d2277d5634f240c2b6764d97